### ISSUE TITLE
Donate Page

### ISSUE THREAD

**Issue by RowdyChildren** [CodeForCharlotte/codeforcharlotte.org#104]
_From @Jillzey on July 4, 2015 17:21_

Duplicate a similar donate page to what we have but connect it directly to the CFC bank account since we no longer need CFA to accept payment (unless 501c3 is required)

_Copied from original issue: CodeForCharlotte/laddrdocs#14_

**Comment by RowdyChildren** [comment]
I agree, when we get 501(c)3 status we should have our own donate page, technology wise are we looking towards PayPal or rolling our own solution with something pike stripe?

**Comment by Jillzey** [comment]
Until this is sorted, we will continue to leave up the CFA link and have put it on the leadership agenda to decide how we want to take payment.

**Comment by RowdyChildren** [comment]
Removing this from the Milestone for Laddr Launch

**Comment by RowdyChildren** [comment]
before we do this we need to implement SSL
